### ISSUE TITLE
bad path when template is used in combination with with_first_found

### ISSUE THREAD

**Issue by drzraf** [sbitio/ansible-munin#5]
until this issue is solved/released <https://github.com/ansible/ansible/issues/6109>

**Comment by drzraf** [comment]
```diff
diff --git a/tasks/config.yml b/tasks/config.yml
index 1e85ad5..1162689 100644
--- a/tasks/config.yml
+++ b/tasks/config.yml
@@ -16,8 +16,8 @@
 - name: Ensure munin-node.conf
   template: src={{ item }} dest={{ munin_node_file_conf }} backup=no owner=root group=root mode=0644
   with_first_found:
-    - "munin-node.conf.{{ ansible_os_family }}.j2"
-    - "munin-node.conf.default.j2"
+    - "../templates/munin-node.conf.{{ ansible_os_family }}.j2"
+    - "../templates/munin-node.conf.default.j2"
   when: munin_role_node == true
 
 - name: Ensure custom plugins directory
```

**Comment by jonhattan** [comment]
It was fixed in https://github.com/ansible/ansible/issues/8879 and available since 1.7.2
